topic2/src/main/java/cartService/CartService.java

```diff
@@ -0,0 +1,20 @@
+package cartService;
+import java.util.LinkedList;
+
+public interface CartService {
+	// Returns the list of items in the cart
+	public LinkedList<Item> getItems();
```

Comment:
Not need to specific public since you are declaring the method inside a interface.